### DISCUSSION TITLE
How do I make a advanced level system

### DISCUSSION THREAD

**Discussion by DotWith** [HaxeFlixel/flixel#2376] [Q&A]
Sorry i'm new to flixel.

I would like to create a level system that stores the background, ground, and etc.

Something like this:
``` xml
<?xml version="1.0" encoding="utf-8"?>
<level>
     <image name="background" src="sky" pos="10, 53" />
     <image name="ground" src="grass" pos="29, 32" />
</level>
```

Yes, I tried tile creator thingie but It's not what i'm trying to achieve.

**Comment by RichardBray** [comment]
This looks a bit like .tmx from Tiled map editor. Flixel supported Tiled map editor which produces XML files for levels, is that the kind of thing your looking for?

**Reply by DotWith** [reply]
Yes, I want something like that, but I want something that doesn't tile anything.

**Reply by RichardBray** [reply]
hmm, I think you might have to roll your own option. There is a way to parse XML in Haxe
https://haxe.org/manual/std-Xml-parsing.html

I guess you can get it to load in the sprite based on the keyword 🤷
